Modules/System/SharePoint/src/model/folder/SharePointFolder.Table.al

```diff
@@ -0,0 +1,81 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about folder entity.
+/// </summary>
+table 9106 "SharePoint Folder"
+{
+    Access = Public;
+    DataClassification = SystemMetadata;
+    Caption = 'SharePoint Folder';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; "Unique Id"; Guid)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Unique Id';
+        }
+
+        field(2; Name; Text[250])
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Title';
+        }
+
+        field(3; Created; DateTime)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Created';
+        }
+
+        field(4; "Item Count"; Integer)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Item Count';
+        }
+
+        field(5; "Exists"; Boolean)
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Exists';
+        }
+
+        field(6; "Server Relative Url"; Text[2048])
+        {
+            DataClassification = SystemMetadata;
+            Caption = 'Server Relative Url';
+        }
+
+        field(101; OdataId; Text[2048])
```

Comment:
Do these field ids have to be in the 100s?

Reply:
renumbered